src/v0/util/index.js

```diff
@@ -1894,5 +1912,6 @@ module.exports = {
   validatePhoneWithCountryCode,
   getEventReqMetadata,
   isHybridModeEnabled,
-  getEventType
+  getEventType,
+  formatValues
```

Comment:
Outside the context of `handleMetadataForValue,` I don't see it's needed. So, let's not export it.

Reply:
The main reason to export this is to make sure it is testable individually. And also I see it might be needed in the future may be for a few fields we might not need to call `constructPayload` or `handleMetadataForValue` function